src/containers/PackingOverview/RemindersPage/RemindersPage.js

```diff
@@ -17,60 +19,139 @@ const RemindersPage = props => {
   const [alertDisplay, setAlertDisplay] = useState(false);
   const [itemInput, setItemInput] = useState("");
+  const [todos, setTodos] = useState([]);
+  const [todoDisplay, setTodoDisplay] = useState(false);
+  const [todoInput, setTodoInput] = useState("");
 
   useEffect(() => {
-    console.log("lists", lists);
+    console.log(lists, "lists");
+    console.log("selected", selectedList);
     if (!lists.length) return;
     for (let list of lists) {
       getList(list);
     }
   }, [lists]);
 
-  const createList = () => {
-    if (lists.list_type === "Todo List" || "Shopping List") {
-      setAlertDisplay(true);
-    } else {
-      axios({
-        method: "post",
-        url: "http://localhost:5000/todolist/",
-        data: {
-          name: "",
-          trip_id,
-          list_type: selectedList
-        }
+  useEffect(() => {
+    console.log(shoppingList, "s list");
+    console.log(shoppingListId, "s id");
+    console.log(todoList, "tdl");
+    console.log(todoListId, "tdl id");
+  }, []);
+
+  useEffect(() => {
+    axios({
+      method: "get",
+      url: BASE_URL + "/todolist/todo/" + lists.todolist_id
+    })
+      .then(data => {
+        console.log(data);
+        setTodos(data);
       })
-        .then(res => {
-          console.log("list created");
-          updateLists();
-        })
-        .catch(err => {
-          console.log(err);
-        });
+      .catch(err => {
+        console.log(err);
+      });
+  }, [todoList, shoppingList]);
+
+  const createList = () => {
+    for (let i = 0; i < lists.length; i++) {
+      if (lists[i].list_type === selectedList) {
+        setAlertDisplay(true);
+        return;
+      }
     }
+    axios({
+      method: "post",
+      url: "http://localhost:5000/todolist/",
+      data: {
+        name: "",
+        trip_id,
+        list_type: selectedList
+      }
+    })
+      .then(res => {
+        console.log("list created");
+        updateLists();
+      })
+      .catch(err => {
+        console.log(err);
+      });
   };
 
   const getList = list => {
+    const { list_type, todolist_id } = list;
+
     axios({
       method: "get",
-      url: "http://localhost:5000/todolist/" + list.todolist_id + "/all"
+      url: "http://localhost:5000/todolist/" + todolist_id + "/all"
     })
       .then(({ data: listData }) => {
-        if (list.list_type === "Shopping List") setShoppingList(listData);
-        else setTodoList(listData);
+        if (list_type === "Shopping List") {
+          setShoppingList(listData);
+          setShoppingListId(todolist_id);
+        } else {
+          setTodoList(listData);
+          setTodoListId(todolist_id);
+        }
       })
       .catch(err => {
         console.log(err);
       });
   };
 
-  const handleOnChange = e => {
+  const handleItemInputChange = e => {
     setItemInput(e.target.value);
   };
 
+  const handleTodoInputChange = e => {
+    setTodoInput(e.target.value);
+  };
+
+  const addTodo = e => {
```

Comment:
This is also something you could probably split out. The `axios` logic isn't relevant to the `RemindersPage`, so it shouldn't be here.

Reply:
Unfortunately, I have to add these to the backlog due to time constraints. I feel the impact of this though so I'll be mindful of it going forward.